src/shared-components/FetchError.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import {FetchError} from './FetchError'
+
+describe('FetchError', () => {
+    it('toMatchSnaphoot', () => {
+        const wrapper = shallow(<FetchError error='Errror message'></FetchError>)
+        expect(wrapper).toMatchSnapshot()
+    })
+})
```

Comment:
@Yagumi please add new line at end of file and change `<FetchError error='Errror message'></FetchError>` to `<FetchError error='Errror message'/>`

Reply:
fixed